cmd/demo.yaml

```diff
@@ -61,6 +61,22 @@ meters:
         if (state.batterySoc > 90) state.batterySoc = 10;
         state.batterySoc;
 
+  - name: battery2
```

Comment:
Warum demo überhaupt ändern, noch dazu inkonsistent (nur 1xCapa bei 2 Batterien)?

Reply:
Sollte beide Möglichkeiten zeigen.
Ist ja schließlich optional.

Reply:
Nimms bitte raus, Möglichkeiten gehören in die dist.yaml, ich schiebs da rein